xml-conduit/Text/XML/Stream/Parse.hs

```diff
@@ -580,14 +604,15 @@ skipSpace = skipWhile isXMLSpace
 --
 -- in <http://www.w3.org/TR/2008/REC-xml-20081126/#sec-common-syn>.
 isXMLSpace :: Char -> Bool
-isXMLSpace ' ' = True
-isXMLSpace '\t' = True
-isXMLSpace '\r' = True
-isXMLSpace '\n' = True
+isXMLSpace '\x20' = True
+isXMLSpace '\x09' = True
+isXMLSpace '\x0D' = True
+isXMLSpace '\x0A' = True
```

Comment:
I changed these to hex literals so that it exactly matches the spec laid out above.